### PR TITLE
Add resend confirmation link view

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

This PR adds the resend link view for when users click on a a sunrise message link that has expired. If the message hasn't been accepted or completed, but the link has expired users will have the ability to request a new confirmation link to initiate the workflow to view the data.

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/image/51353636?key=34bc161f457c32db0185ae1e990831e6

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: pkg/web/templates/partials/sunrise/verification.html [hunk 1]
```diff
@@ -1,17 +1,34 @@
-<div id="verify-code" class="mx-auto max-w-4xl">
+<!-- If the confirmation link is valid and hasn't expired, display the email verification -->
```
Comment (daniellemaxwell) [inline pkg/web/templates/partials/sunrise/verification.html:1]: This is how I believe the workflow will work. A user will click a link and htmx will trigger this partial to load. If the confirmation link is valid and hasn't expired users will see the verify email form. If the link has expired, the user see the resend confirmation link view.

File: pkg/web/templates/partials/sunrise/verify_code.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div id="verify-code" class="mx-auto max-w-4xl">
```
Comment (daniellemaxwell) [inline pkg/web/templates/partials/sunrise/verify_code.html:1]: Once a user clicks the submit button in the verify email form, this partial would be loaded into the empty div in verify.html.

File: pkg/web/static/brokenlink.svg
Comment (bbengfort) [inline pkg/web/static/brokenlink.svg:1]: :-) nice icon!

File: pkg/web/templates/partials/sunrise/verification.html (hunk 1 above)
Comment (bbengfort) [reply pkg/web/templates/partials/sunrise/verification.html:1]: That makes sense to me! We can either render a different partial in the back-end or use HX-Redirect, but either way, the front-end stays the same.

**Review by bbengfort — APPROVED** [review]
Looks great! I get where you're going with the workflow and it makes sense to me!

**Pull request merged**